src/_common/LabeledText.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const LabeledText = ({id, label, value = 'N/A'}) => (
+    <tr id={id} className={'name-val-pair'}>
```

Comment:
Flexbox is more powerful than tables.
I would have created this as a pair of label elements, or span elements, then style them.